### ISSUE TITLE
Potential typo on timesync Galaxy docs

### ISSUE THREAD

**Issue by falcon78921** [linux-system-roles/timesync#44]
I'm not 100% sure if this is the correct repo to report this issue to. If I need to be directed somewhere else, please let me know.

On the `timesync` Galaxy page README (https://galaxy.ansible.com/linux-system-roles/timesync), I believe there's a typo at the bottom of the example playbook reference.

Specifically, here:

~~~
  roles:
    - timesync
~~~

When I execute this playbook using the preceding role definition, I get an error saying that `timesync` is not a valid role. However, the playbook correctly executes when defining it like this:

~~~
  roles:
    - linux-system-roles.timesync
~~~

I think `timesync` needs to be switched to `linux-system-roles.timesync`. Thanks! :)

**Comment by pcahyna** [comment]
Thanks for the report, we should cut a new release... this is fixed in the GitHub master, see https://github.com/linux-system-roles/timesync#example-playbook . For now please use a GitHub checkout.

**Comment by pcahyna** [comment]
This is fixed in 1.0.2, already published on Galaxy.
